internal_transfer_with_agreed_amount/tests/test_internal_transfer_with_agreed_amount.py

```diff
@@ -7,7 +7,12 @@ def setUp(self):
         self.currency_usd = self.env.ref("base.USD")
```

Comment:
Typo in commit description:

readibility -> readability

Reply:
Fixed.